### PR TITLE
1170: Notify bot reposts commit comments if another notifier fails

### PR DESCRIPTION
**Pull request by erikj79**
Prevent the CommitCommentNotifier from posting the exact same comment twice.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Issue
 * [SKARA-1170](https://bugs.openjdk.java.net/browse/SKARA-1170): Notify bot reposts commit comments if another notifier fails

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/skara pull/1218/head:pull/1218` \
`$ git checkout pull/1218`

Update a local copy of the PR: \
`$ git checkout pull/1218` \
`$ git pull https://git.openjdk.java.net/skara pull/1218/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1218`

View PR using the GUI difftool: \
`$ git pr show -t 1218`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/skara/pull/1218.diff">https://git.openjdk.java.net/skara/pull/1218.diff</a>

</details>

### CODE REVIEW

File: bots/notify/src/test/java/org/openjdk/skara/bots/notify/comment/CommitCommentNotifierTests.java [hunk 1]
```diff
@@ -165,4 +165,61 @@ void testCommitCommentWithIssues(TestInfo testInfo) throws IOException {
...
         }
     }
+
+    /**
+     * Test that the CommitCommentNotifier never repeates the exact same comment
+     */
+    @Test
+    void testNoRepeatedCommitComment(TestInfo testInfo) throws IOException {
+        try (var credentials = new HostCredentials(testInfo);
+             var tempFolder = new TemporaryDirectory()) {
```
Comment (kevinrushforth) [inline bots/notify/src/test/java/org/openjdk/skara/bots/notify/comment/CommitCommentNotifierTests.java:175]: Minor: indentation is different from below lines.

**Review by kevinrushforth — APPROVED** [review]
Looks good.

File: bots/notify/src/test/java/org/openjdk/skara/bots/notify/comment/CommitCommentNotifierTests.java (hunk 1 above)
Comment (erikj79) [reply bots/notify/src/test/java/org/openjdk/skara/bots/notify/comment/CommitCommentNotifierTests.java:175]: Yes, that seems to be a recurring style choice in the codebase. The contents of the try () is lined up with the start of the opening parentheses.

**Pull request closed without merging**